db/migrate/20220527120826_add_index_on_category_slug_and_locale.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexOnCategorySlugAndLocale < ActiveRecord::Migration[6.1]
+  def change
+    add_column :categories, :slug, :string, null: false, default: 'common'
+    add_index :categories, [:slug, :locale], unique: true
```

Comment:
I think the uniqueness should be on [portal_id, slug, locale] not on [slug, locale]